src/org/exist/xmldb/LocalResourceSet.java

```diff
@@ -87,6 +87,13 @@ public void addResource(final Resource resource) throws XMLDBException {
         resources.add(resource);
     }
 
+    @Override
```

Comment:
why is this needed?

Reply:
The "official" API has this, it was really the only difference apart from `int` vs `long` for some sizes.